es5-persistence/src/main/java/com/netflix/conductor/dao/es5/index/ElasticSearchDAOV5.java

```diff
@@ -133,20 +132,11 @@
     }
 
     @Inject
-    public ElasticSearchDAOV5(Client elasticSearchClient, Configuration config, ObjectMapper objectMapper) {
+    public ElasticSearchDAOV5(Client elasticSearchClient, ElasticSearchConfiguration config, ObjectMapper objectMapper) {
         this.objectMapper = objectMapper;
         this.elasticSearchClient = elasticSearchClient;
-        this.indexName = config.getProperty("workflow.elasticsearch.index.name", null);
-
-        try {
-
-            initIndex();
-            updateIndexName(config);
-            Executors.newScheduledThreadPool(1).scheduleAtFixedRate(() -> updateIndexName(config), 0, 1, TimeUnit.HOURS);
-
-        } catch (Exception e) {
-            logger.error(e.getMessage(), e);
-        }
+        this.indexName = config.getIndexName();
+        this.logIndexPrefix = config.getProperty("workflow.elasticsearch.tasklog.index.name", "task_log");
```

Comment:
Consider moving strings to local constants. It might help in readability and further changes will create single line diff.

Reply:
Sounds good.
Note this PR just moved some code around in order to allow setting up the indexDao (if necessary). This improvement can actually be done over the current code (branch 2.0). Having said that, I think it will just improve the readability as you said, so will update the PR probably today 👍